plugins/quick-brick-inplayer-login/runInBackground.js

```diff
@@ -1,7 +1,11 @@
+import * as R from "ramda";
+
 import { getLoginProtocol } from "./src/Services/loginPluginProtocol";
 
 export const runInBackground = async (item, callback, configuration, presentUI) => {
-  const { isLoggedIn } = await getLoginProtocol();
+  const in_player_environment = R.pathOr(undefined, ['configuration', 'in_player_environment'], configuration)
```

Comment:
R.path should be enough in this case.
```suggestion
  const in_player_environment = R.path(['configuration', 'in_player_environment'], configuration)
```